admin/includes/header.php

```diff
@@ -130,7 +130,7 @@
     // display the "check for updated version" button.  The button link should be the current page and all params
     $url = zen_href_link(basename($PHP_SELF), zen_get_all_get_params(array('vcheck'), 'SSL'));
     $url .= (strpos($url, '?') > 5 ? '&' : '?') . 'vcheck=yes';
-    if ($zv_db_patch_ok == true || $version_check_sysinfo == true) $new_version .= '<a href="' . $url . '">' . '<input type="button" class="btn btn-link" value="' . TEXT_VERSION_CHECK_BUTTON . '"/></a>';
+    if ($zv_db_patch_ok == true || $version_check_sysinfo == true) $new_version .= '<a href="' . $url . '">' . TEXT_VERSION_CHECK_BUTTON . '</a>';
```

Comment:
Why not leave it as a button
```
<a href="' . $url . '" role="button" class="btn btn-link">' . TEXT_VERSION_CHECK_BUTTON . '</a>
```

Reply:
you cannot nest an input inside an <a tag
And it is only a text link.

Reply:
You are right about the nesting, but look at the line I posted above, The class makes a button with css.

Reply:
That validates. 
I didn't bother to maintain it as a button as it is not formatted as a button, just as a link that underlines, didn't think about the accessibility aspect.
Corrected.